cmd/cloudflared/main.go

```diff
@@ -134,11 +134,22 @@ To determine if an update happened in a script, check for error code 11.`,
 		{
 			Name: "version",
 			Action: func(c *cli.Context) (err error) {
+				if c.Bool("short") {
+                    fmt.Println(strings.Split(c.App.Version, " ")[0])
```

Comment:
Could you run `go fmt ./...` on the repo to make sure this is formatted properly.

Reply:
Just did that :)